pkg/infrastructure/gcp/clusterapi/ignition.go

```diff
@@ -18,9 +18,9 @@ import (
 // editIgnition attempts to edit the contents of the bootstrap ignition when the user has selected
 // a custom DNS configuration. Find the public and private load balancer addresses and fill in the
 // infrastructure file within the ignition struct.
-func editIgnition(ctx context.Context, in clusterapi.IgnitionInput) ([]byte, error) {
+func editIgnition(ctx context.Context, in clusterapi.IgnitionInput) ([]byte, []byte, error) {
```

Comment:
Nit: Do we think that we will have other ignition data to edit in the future? It's certainly possible right? Maybe we should consider a struct returned rather than more return values.